packages/contracts-bedrock/src/L1/RoundRobinLeaderElection.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
Can we use the latest version? I see some contracts in the repo use 0.8.19.

Reply:
Sure, done in 0fc911ee6f9866702cb21c86b0091410cf4f17cd.